docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -1529,7 +1529,20 @@ Query Parameters:
 
     Request Headers:
 
--   **X-Registry-Auth** – base64-encoded AuthConfig object
+-   **X-Registry-Auth** – base64-encoded AuthConfig object, containing either login information, or a token
+    - Credential based login:
+
+    {
+        "username": "jdoe",
+        "password": "secret",
+        "email": "jdoe@acme.com",
+    }
+
+    - Token based login:
+
+    {
+        "registrytoken": "9cbaf023786cd7..."
+    }
```

Comment:
the code examples probably need to be indented more (4 spaces) to show up as code blocks, see the rendered page;

<img width="693" alt="screen shot 2015-12-02 at 20 09 12" src="https://cloud.githubusercontent.com/assets/1804568/11541110/bf104634-9930-11e5-8cc8-2796cb01fd76.png">